tests/annotation_tests.ts

```diff
@@ -15,14 +15,14 @@ import { Beam } from '../src/beam';
 import { Bend } from '../src/bend';
 import { ElementStyle } from '../src/element';
 import { Flow } from '../src/flow';
-import { Font, FontStyle, FontWeight } from '../src/font';
 import { Formatter } from '../src/formatter';
 import { ModifierPosition } from '../src/modifier';
 import { Registry } from '../src/registry';
 import { ContextBuilder } from '../src/renderer';
 import { Stave } from '../src/stave';
 import { StaveNote, StaveNoteStruct } from '../src/stavenote';
 import { Stem } from '../src/stem';
+import { Tables } from '../src/tables';
```

Comment:
I know a couple of the test files import Tables, but did we ever decide of Tables is supposed to be internal use, or for public use?

If it is for public use, then we should be much more careful with the design of the API. Right now, Tables is sort of a kitchen sink of useful object dictionaries that are used by our internal classes.

I think Tables was intended to be internal, so that's why we had a `flow.ts` class that made some of the internal Tables data public.

So we should be careful when we directly access Tables in our tests, because the test code is how some developers learn how to use the public APIs.

Reply:
same as above, please leave it to another PR